lib/rules/yield-effects.js

```diff
@@ -25,6 +25,16 @@ module.exports = {
     var inGeneratorDepth = 0
     var effectLocalNames = []
     var effectImportedNames = []
+    function inGeneratorDepthIncrement(node) {
```

Comment:
Please add an empty line before the function declaration and please rename the functions to `enterFunction` and `exitFunction` respectively.

Reply:
Sorry for the long time of waiting. I've added these suggestions.

Reply:
no worries. I'll merge and build a new version when I have time myself ;)